noise/README.md

```diff
@@ -243,9 +247,8 @@ spec][peer-id-spec-signing-rules]. The data to be signed is the UTF-8 string
 `noise-libp2p-static-key:`, followed by the Noise static public key, encoded
 according to the rules defined in [section 5 of RFC 7748][rfc-7748-sec-5].
 
-The `data` field contains the "early data" provided to the Noise module when
-initiating the handshake, if any. The structure of this data is opaque to
-noise-libp2p and is defined in the connection establishment specs.
+The `extensions` field contains Noise extensions and is described in
+[noise-extensions].
```

Comment:
I don't think this is a valid GitHub Markdown link. At least it does not resolve in a hyperlink when I view the markdown file directly.

Reply:
Fixed.